ttrpc-codegen/Cargo.toml

```diff
@@ -15,7 +15,7 @@ readme = "README.md"
 [dependencies]
 # lock home to avoid conflict with latest version
 home = "=0.5.9"
-protobuf-support = "3.2.0"
-protobuf = { version = "2.27.1" }
-protobuf-codegen = "3.5.1"
-ttrpc-compiler = "0.7.0"
+protobuf-support.workspace = true
```

Comment:
In my opinion, This way of writing is not as intuitive as the original one.

Reply:
Would you find this more intuitive?
```toml
protobuf-support = { workspace = true }
```